docs/docsite/rst/conf.py

```diff
@@ -18,6 +18,17 @@
 
 import sys
 import os
+from pathlib import Path
+
+
+DOCS_ROOT_DIR = Path(__file__).parent.resolve()
+
+
+# Make in-tree extension importable in non-tox setups/envs, like RTD.
+# Refs:
+# https://github.com/readthedocs/readthedocs.org/issues/6311
+# https://github.com/readthedocs/readthedocs.org/issues/7182
+sys.path.insert(0, str(DOCS_ROOT_DIR.parent / '_ext'))
```

Comment:
Just wanted to say I've been looking into this bit and it's really quite neat. Hats off.

Reply:
Oh, I think it's in the default Sphinx skeleton it generates… With the addition of bringing in pathlib. But when people clean up unneeded things, the defaults get lost.
https://github.com/sphinx-doc/sphinx/blob/8962398/sphinx/templates/quickstart/conf.py.jinja#L7-L11

Reply:
@oraNod apparently you already used this approach two years ago: https://github.com/ansible/awx/pull/14607/files#diff-7f57eeb7558bfb5b6456b5d7d19497732f9b4653a42a04912bc48d9e3b86496cR8